Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -54,12 +54,10 @@ public enum CursorStateEnum
         public CursorStateEnum CursorState { get { return cursorState; } }
         private CursorStateEnum cursorState = CursorStateEnum.None;
 
-#pragma warning disable 0649
```

Comment:
Rather than disabling the warning, we made the field "protected" which has an added benefit that if a child class introduces a new SerializedField with the same name, the compiler will help us catch it, rather than leaving Unity to silently ignore the issue and leave us developers confused.